src/ci/docker/host-x86_64/dist-x86_64-linux/build-gccjit.sh

```diff
@@ -1,19 +1,22 @@
 #!/usr/bin/env bash
 
+GIT_REPO="https://github.com/rust-lang/gcc"
+
+# This commit hash needs to be updated to use a more recent gcc fork version.
+GIT_COMMIT="78dc50f0e50e6cd1433149520bd512a4e0eaa1bc"
+
 set -ex
 
 cd $1
 
 source shared.sh
 
 # Setting up folders for GCC
-git clone https://github.com/antoyo/gcc gcc-src
-cd gcc-src
-# This commit hash needs to be updated to use a more recent gcc fork version.
-git checkout 78dc50f0e50e6cd1433149520bd512a4e0eaa1bc
+curl -L "$GIT_REPO/archive/$GIT_COMMIT.tar.gz" |
```

Comment:
We should probably also [`retry`](https://github.com/rust-lang/rust/blob/35dfc67d94c47a6c6ae28c46e7dc1c547f772485/src/ci/shared.sh#L12) the download if it fails for some reason.
```suggestion
retry curl -L "$GIT_REPO/archive/$GIT_COMMIT.tar.gz" |
```

Reply:
The issue here is not so much the download but the transient DNS failures, they happened in the past, even for `github.com` (can't find a example right now).

Sure, we may want to download first to disk; I didn't look to closely at the retry function.

Reply:
I wouldn't worry about it. The whole docker build is already retried 5 times IIRC. This is maybe nice cleanup in the future but isn't worth it for now.